### PR TITLE
Fixed orgs using d on large screens

### PR DESCRIPTION
**Pull request by JackStouffer**
Also added prefixed css rules

### CODE REVIEW

**Comment by wilzbach** [comment]
Before:

image


After:

image

File: css/style.css [hunk 1]
```diff
@@ -1425,12 +1425,29 @@ body#Home #content > .intro .download .btn.action:hover
 
 .org_list
 {
+    max-width: 1400px;
+    margin: 0 auto;
+    padding: 10px 0;
+
+    display: -webkit-box;
+    display: -ms-flexbox;
     display: flex;
```
Comment (wilzbach) [inline css/style.css]: `flex` is a standard:

https://developer.mozilla.org/en-US/docs/Web/CSS/flex
https://www.w3.org/TR/css-flexbox-1

File: css/style.css [hunk 2]
```diff
@@ -1425,12 +1425,29 @@ body#Home #content > .intro .download .btn.action:hover
...
+
+    -webkit-box-align: center;
+    -ms-flex-align: center;
     align-items: center;
+
+    -webkit-box-orient: horizontal;
+    -webkit-box-direction: normal;
+    -ms-flex-direction: row;
```
Comment (wilzbach) [inline css/style.css]: It's a standard now: https://developer.mozilla.org/en-US/docs/Web/CSS/flex-direction

File: css/style.css [hunk 3]
```diff
@@ -1425,12 +1425,29 @@ body#Home #content > .intro .download .btn.action:hover
...
     flex-direction: row;
+
+    -webkit-box-pack: space-evenly;
+    -ms-flex-pack: space-evenly;
     justify-content: space-evenly;
+
+    -ms-flex-wrap: wrap;
```
Comment (wilzbach) [inline css/style.css]: Standard: https://www.w3.org/TR/css-flexbox-1/#propdef-flex-wrap

**Comment by JackStouffer** [comment]
We hit an extra 10% of the browser market with the prefixes https://caniuse.com/#feat=flexbox

**Comment by wilzbach** [comment]
Here's what I see:

image

Those are all platforms we don't care about.
Also we use `display: flex` at other places too without worrying about it, e.g. 

https://github.com/dlang/dlang.org/pull/1437 (since 1 1/2 years)

**Comment by JackStouffer** [comment]
Fixed

**Pull request merged**